user/feh/template.py

```diff
@@ -0,0 +1,44 @@
+pkgname = "feh"
+pkgver = "3.10.3"
+pkgrel = 0
+build_style = "makefile"
+make_build_args = [
+    "exif=1",
+    "help=1",
+    "inotify=1",
+    "magic=1",
+    "stat64=1",
+]
+makedepends = [
+    "file-devel",
+    "imlib2-devel",
+    "libcurl-devel",
+    "libexif-devel",
+    "libpng-devel",
+    "libx11-devel",
+    "libxinerama-devel",
+    "libxt-devel",
+]
+checkdepends = ["perl", "perl-test-command", "mandoc"]
+pkgdesc = "Fast and light image viewer"
+maintainer = "daringcuteseal <daringcuteseal@gmail.com>"
+license = "MIT"
+url = "https://feh.finalrewind.org"
+source = f"{url}/feh-{pkgver}.tar.bz2"
+sha256 = "5426e2799770217af1e01c2e8c182d9ca8687d84613321d8ab4a66fe4041e9c8"
+# flags required but not passed otherwise
+tool_flags = {
+    "CFLAGS": [
+        '-DPREFIX="/usr"',
+        f'-DPACKAGE="{pkgname}"',
+        f'-DVERSION="{pkgver}"',
+    ]
+}
```

Comment:
you want make_use_env = True above instead of these. currently the build is broken since none of the libraries are linked (e.g. curl, exif, etc)